src/parser/core/modules/__tests__/PrecastStatus.ts

```diff
@@ -0,0 +1,477 @@
+// region CodeUnderTest
```

Comment:
I'm not a vast fan of regions, but I can see their use. If you're going to add them, can you use `// #region` and `// #endregion` so other editors pick them up properly?